src/async/stompest/async/client.py

```diff
@@ -135,14 +135,16 @@ def session(self):
     # STOMP commands
     #
     @defer.inlineCallbacks
-    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None):
-        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None)
+    def connect(self, headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None):
+        """connect(headers=None, versions=None, host=None, heartBeats=None, connectTimeout=None, connectedTimeout=None, certKey=None, privateKey=None)
```

Comment:
Thanks @cjrh for the review!
I would love to make this work with `SSLContext`. The two issues I ran into:

1. Currently we use Twisted's [clientFromString()](https://twistedmatrix.com/documents/current/api/twisted.internet.endpoints.clientFromString.html), and that requires that the SSL cert/key pair be in files on disk, rather than in-memory `ssl` structures. Maybe we could stop using clientFromString() and use [SSL4ClientEndpoint()](https://twistedmatrix.com/documents/current/api/twisted.internet.endpoints.SSL4ClientEndpoint.html) instead? (as in the [walkthrough](https://twistedmatrix.com/documents/current/core/howto/ssl.html#client-with-certificates)). (That connection string is getting pretty complex regardless.)

2. I can load the certfile and keyfile into the context object with `load_cert_chain()`, but I cannot find a trivial way to get them back out :)

What do you think?